yb-voyager/cmd/exportData.go

```diff
@@ -168,6 +168,12 @@ func exportData() bool {
 			if err != nil {
 				utils.ErrExit("failed to create trigger file after data export: %v", err)
 			}
+			if isTargetDBExporter(exporterRole) {
```

Comment:
i guess this will work for fall back switchover also.
In case the title of PR can be modified: `
Deleting the yb cdc stream id after fall-forward//fall-back switchover`

Reply:
Yes, thanks!